src/CitMovie.Business/Managers/GenreManager.cs

```diff
@@ -0,0 +1,28 @@
+using CitMovie.Data.Repositories;
+using CitMovie.Models.DTOs;
+
+namespace CitMovie.Business.Managers;
+
+public class GenreManager : IGenreManager
+{
+    private readonly IGenreRepository _genreRepository;
+
+    public GenreManager(IGenreRepository genreRepository)
+    {
+        _genreRepository = genreRepository;
+    }
+    public async Task<IList<GenreDto>> GetAllGenresAsync(int page, int pageSize)

```

Comment:
This should be IEnumerable right? And not IList. We should try and use the least specific type.

This [article ](https://www.linkedin.com/pulse/what-difference-ienumerable-vs-ilist-icollection-kushal-developer/)explains it well I think